### PR TITLE
docs: known issues 5.1

### PR DESCRIPTION
**Pull request by coury-clark**
Adding known issue for repo permission problem in 5.1

## Test plan
N/A just changelog

### CODE REVIEW

**Review by marcleblanc2 — CHANGES_REQUESTED** [review]
To clarify, does this only impact instances using the explicit perms API?

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ All notable changes to Sourcegraph are documented in this file.
 
 > **Note**: As of 5.1.0, Sourcegraph OSS is removed and has been relicensed to Sourcegraph Enterprise. A blog post with much more detail will come soon!
 
+### Known issues
+
+- There is an issue with repository permissions not being enforced on code hosts that do not support the `authorization` field.
```
Comment (coury-clark) [inline CHANGELOG.md:40]: There is an issue with repository permissions not being enforced on Sourcegraph instances that are configured to use both explicit permissions and code hosts that do not support the `authorization` field.

Comment (coury-clark) [inline CHANGELOG.md:40]:
```suggestion
There is an issue with repository permissions not being enforced on Sourcegraph instances that are configured to use both explicit permissions and code hosts that do not support the authorization field.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ All notable changes to Sourcegraph are documented in this file.
 
 > **Note**: As of 5.1.0, Sourcegraph OSS is removed and has been relicensed to Sourcegraph Enterprise. A blog post with much more detail will come soon!
 
+### Known issues
+
+There is an issue with repository permissions not being enforced on Sourcegraph instances that are configured to use both explicit permissions and code hosts that do not support the authorization field. Customers using the explicit permissions API are advised to upgrade to v5.1.1 directly.
```
Comment (coury-clark) [inline CHANGELOG.md:40]:
```suggestion
There is an issue with Sourcegraph instances configured to use explicit permissions using permissions.userMapping in Site configuration, where repository permissions are not enforced. Customers using the explicit permissions API are advised to upgrade to v5.1.1 directly.
```

**Review by marcleblanc2 — APPROVED** [review]
Great work @coury-clark and team!

**Pull request merged**